File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/Form/Programming/ActivatingForm.cs
#region Using

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;

using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms;

#endregion

namespace CompanionKit.Controls.Form.Programming
{
    public partial class ActivatingForm : global::Gizmox.WebGUI.Forms.Form
    {
        public ActivatingForm()
        {
            InitializeComponent();
        }

        /// <summary>
        /// Handles Click event of the button.
        /// Closes the Form.
        /// </summary>
        private void mobjButton_Click(object sender, EventArgs e)
        {
            this.Close();
        }

        /// <summary>
        /// Handles Activated event of the form.
        /// Updates text of the form label.
        /// </summary>
        private void ActivatingForm_Activated(object sender, EventArgs e)
        {
            this.mobjLabel.Text = "This form is Activated!";
        }

        /// <summary>
        /// Handles Deactivate event of the form.
        /// Updates text of the form label.
        /// </summary>
        private void ActivatingForm_Deactivate(object sender, EventArgs e)
        {
            this.mobjLabel.Text = "This form is Deactivated!";
        }
    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/Form/Programming/ActivatingForm.vb
Namespace CompanionKit.Controls.Form.Programming
	Public Class ActivatingForm

        ''' <summary>
        ''' Handles Click event of the button.
        ''' Closes the Form.
        ''' </summary>
        ''' <param name="sender"></param>
        ''' <param name="e"></param>
        ''' <remarks></remarks>
        Private Sub mobjButton_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles mobjButton.Click
            MyBase.Close()
        End Sub

        ''' <summary>
        ''' Handles Activated event of the form.
        ''' Updates text of the form label.
        ''' </summary>
		Private Sub ActivatingForm_Activated(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Activated

            Me.mobjLabel.Text = "This form is Activated!"

		End Sub

        ''' <summary>
        ''' Handles Deactivate event of the form.
        ''' Updates text of the form label.
        ''' </summary>
		Private Sub ActivatingForm_Deactivate(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Deactivate

            Me.mobjLabel.Text = "This form is Deactivated!"

		End Sub
	End Class
End Namespace
